### ISSUE TITLE
What is the best config (params.ini)?

### ISSUE THREAD

**Issue by msobhan69** [Conchylicultor/DeepQA#157]
I open this issue to share your ideas and experiences on setting the values of parameters in params.ini.
I want to train on OpenSubs dataset with following settings:

[General]
version = 0.5
globstep = 800000
watsonmode = False
autoencode = False
corpus = opensubs
[Dataset]
datasettag = 
maxlength = 30
filtervocab = 1
skiplines = False
vocabularysize = 0
[Network]
hiddensize = 512
numlayers = 2
softmaxsamples = 0
initembeddings = true
embeddingsize = 300
embeddingsource = fastText.bin
[Training (won't be restored)]
learningrate = 0.002
batchsize = 256
dropout = 0.9

can you share your settings and sample results?

**Comment by msobhan69** [comment]
my results for now:
![screenshot-2017-10-30 tensorboard](https://user-images.githubusercontent.com/26247452/32156965-b4a57a02-bd55-11e7-9fc0-ec5edfee95eb.png)

Perplexity: 19
loss: 2.9

Welcome to the interactive mode, here you can ask to Deep Q&A the sentence you want. Don't have high expectation. Type 'exit' or just press ENTER to quit the program. Have fun.
Q: hi
A: Hi

Q: What's up?
A: I' m sorry.

Q: Hello
A: Hello

Q: Who are you?
A: I' m a friend.

Q: nice to meet you
A: Visiting the world

Q: glad to see you
A: I' m the one who' s the only one who can' t be

Q: My name is Sobhan
A: And i' m a good man

Q: What is your name?
A: Marcello.

Q: Can you help me, Marcello?
A: No, i' m not.

Q: Why?
A: Because i' m a cop.

Q: where are you?
A: I' m not going to hurt you.

Q: i know
A: I' m not going to let you go

Q: ok
A: Good night

Q: What color is sky?
A: It' s a condom.

Q:  What color is the sky?
A: It' s a monster.

Q: Hello?
A: Hello?

Q: How are you?
A: I' m fine.

Q: How so?
A: I' m not going to be able to get a chance.

Q: What is your job?          
A: I' m not going to be able to get you out of here.

Q: What is the purpose of life?
A: I don' t know.

Q: Are you intelligent?
A: Yes.

Q: Are you a machine?
A: I' m not sure

Q: Are you human?
A: Yes.

Q: What are you?
A: I' m sorry.

Q: Do you plan on taking over the world?
A: I' m not sure.

Q: What the fuck are you doing?
A: I' m not going to hurt you.

Q: le me go
A: I' m not going to be here

Q: let me go
A: God

Q: I don't like you
A: I' m a little too much

Q: do you love me?
A: I love you.

Q: who is bugart
A: Who is the man who' s the only one who can' t see

Q: Who is the president?
A: I don' t know.

Q: Who is president of USA now?
A: I don' t know.

Q: ok
A: Good night

Q: good night
A: Clouds like a bird

Q: Are you ok??
A: I' m fine.

Q: well, Who are you?
A: I' m a little nervous.

Q: Why?
A: Because i' m a cop.

Q: What color is the sea?
A: I don' t know.

Q: What is the sky?
A: A ghost.

Q: What is a ghost?
A: I don' t know.

Q:

**Comment by msobhan69** [comment]
According to the results, I think there are many reasons to this bad results:
1- The OpenSubs dataset is too small
2- The model overfit on the data
3- The model just see the last question and ignore the previous dialogs
4- The model can't do reasoning
Any idea to modify big models like Tacotron for chatbot
